packages/vitest/src/runtime/runners/test.ts

```diff
@@ -44,6 +48,8 @@ export class VitestTestRunner implements VitestRunner {
       if (result)
         await rpc().snapshotSaved(result)
     }
+
+    this.workerState.current = suite.suite
```

Comment:
`current` is the test, it shouldn't reference suites. Maybe it's better to add another property for this?

Reply:
I noticed that when I was investigating, but I still used it as its type is `Task` here https://github.com/vitest-dev/vitest/blob/9abef3d004a9d3c0d61ae81ecffaa248687ff75d/packages/vitest/src/types/worker.ts#L28 

if `current` should be the current test, how about adding a property `currentSuite`?